be/src/storage/version_graph.cpp

```diff
@@ -388,8 +389,10 @@ Status VersionGraph::capture_consistent_versions(const Version& spec_version,
     auto end_vertex_iter = _version_graph.find(spec_version.second + 1);
 
     if (start_vertex_iter == _version_graph.end() || end_vertex_iter == _version_graph.end()) {
-        LOG(WARNING) << "fail to find path in version_graph. "
-                     << "spec_version: " << spec_version.first << "-" << spec_version.second;
+        RATE_LIMIT(LOG(WARNING) << "fail to find path in version_graph. "
```

Comment:
Also need to print the log, otherwise it is useful to trace the problem

Reply:
Done.